pyproject.toml

```diff
@@ -18,15 +18,18 @@ classifiers = [
   "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
   "Framework :: Django",
   "Framework :: Django :: 3.2",
   "Framework :: Django :: 4.0",
+  "Framework :: Django :: 4.1",
+  "Framework :: Django :: 4.2",
 ]
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
-Django = ">=3.0"
-strawberry-graphql = ">=0.139.0"
+Django = ">=3.2,<4.2"
```

Comment:
Hi there,

since we are testing this with django `4.2a1`.. Do we need to add `<4.2` ? Because this for this installation is failing with django `4.2a1

Reply:
You are right! I just published a new version removing the upper bound limit from both django and strawberry

Reply:
Thank you for fast response..